packages/opentelemetry-node/test/spanprocessor-eventprocessor.test.ts

```diff
@@ -0,0 +1,51 @@
+import { NodeTracerProvider } from '@opentelemetry/sdk-trace-node';
```

Comment:
Is this test supposed to be part of this PR?

Reply:
Nope, just forked off the wrong branch 🤦